### PR TITLE
Add `neondatabase/release` team as default reviewers for storage releases

### PR DESCRIPTION
**Pull request by vadim2404**
A group of people who can approve releases will be added as default reviewers for such PRs

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [ ] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

### CODE REVIEW

**Comment by koivunej** [comment]
Regress test failure triaged: https://github.com/neondatabase/neon/issues/3360#issuecomment-1463798463

**Comment by vadim2404** [comment]
> Regress test failure triaged: [#3360 (comment)](https://github.com/neondatabase/neon/issues/3360#issuecomment-1463798463)

I didn't get your comment

**Comment by koivunej** [comment]
> I didn't get your comment

I just happened to notice this zero-code change had a regression test failure, I triaged/investigated and fixed it (it was a flaky test fix).

**Comment by vadim2404** [comment]
> > I didn't get your comment
> 
> I just happened to notice this zero-code change had a regression test failure, I triaged/investigated and fixed it (it was a flaky test fix).

Oh, gotcha. Thanks

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -31,3 +31,4 @@ jobs:
         head: releases/${{ steps.date.outputs.date }}
         base: release
         title: Release ${{ steps.date.outputs.date }}
+        team_reviewers: release
```
Comment (bayandin) [inline .github/workflows/release.yml:34]: Should we create a group first?
I don't see it in https://github.com/orgs/neondatabase/teams

Comment (vadim2404) [reply .github/workflows/release.yml:34]: https://github.com/orgs/neondatabase/teams/release

Here it is

**Review by bayandin — APPROVED** [review]

**Pull request merged**